ee/api/test/test_capture.py

```diff
@@ -202,7 +202,7 @@ def test_partition_key_override(self, kafka_produce):
 
         # Setting up an override via EVENT_PARTITION_KEYS_TO_OVERRIDE should cause us to pass None
         # as the key when producing to Kafka, leading to random partitioning
-        with self.settings(EVENT_PARTITION_KEYS_TO_OVERRIDE=default_partition_key):
+        with self.settings(EVENT_PARTITION_KEYS_TO_OVERRIDE=[default_partition_key]):
```

Comment:
Can we explicitly check that, when there is a db error we still receive the event in Kafka?

Reply:
yup, still need to write tests for the capture endpoint!